RNTester/js/RTLExample.js

```diff
@@ -526,18 +526,19 @@ class RTLExample extends React.Component<any, State> {
     );
   };
 
-  _linearTap = (refName: string, e: Object) => {
+  _linearTap = (e: Object) => {
```

Comment:
i'm actually not sure how this worked previously at all because this function is passed to the onPress of TouchableWithoutFeedback which doesn't pass a first and second argument. 

I guess I understand how you were keeping the behavior the same by just using the object. 

It looks like in the original commit for this file, it used

```
onPress={this._linearTap.bind(this)}
```

which was probably copied and pasted from a different internal file which uses the following for the AnimationBlock:

```
this.linearTap.bind(this, 'linear_ref')
```

And then in 2016 this was cleaned up in this commit: https://github.com/facebook/react-native/commit/e2d84a8221391cb9b405bb64b131e60d46d81172 changing:

```
onPress={this._linearTap.bind(this)}
```

to
```
onPress={this._linearTap}
```

----------------

So, looks like there is a lot of legacy here that probably doesn't make sense anymore. We probably don't want to use the location of the touch as the key because then every touch it would be a different key instead of the intention which is tied to this component instance. 

For the minimal amount of changes, feel free to just go back to using the event as the key, even though it doesn't make sense. That at least means that there are no changes in this PR.

As a separate diff, this could probably be revisited as from my initial glancing it doesn't seem like this is created in a loop or needing to keep track of the toggleStatus for multiple things so a lot of this logic could probably be deleted, but that is probably a change that will require more testing. To unblock, changing this back to the event is reasonable. Sorry for all the churn. I understand this a lot better now.

Reply:
Thanks for the feedbacks, no worries it was interesting 😄.